README.md

```diff
@@ -148,6 +148,32 @@ var args struct {
 arg.MustParse(&args)
 ```
 
+### Custom placeholders
+
```

Comment:
Add a little text here: "Use the placeholder tag to control which placeholder text is used in the usage text"